locales/fr.json

```
@@ -246,9 +246,9 @@
     "user_updated": "L’utilisateur a été modifié",
     "yunohost_already_installed": "YunoHost est déjà installé",
     "yunohost_ca_creation_failed": "Impossible de créer l’autorité de certification",
-    "yunohost_configured": "YunoHost maintenant configuré",
+    "yunohost_configured": "YunoHost est maintenant configuré",
     "yunohost_installing": "L'installation de YunoHost est en cours …",
-    "yunohost_not_installed": "YunoHost n’est pas ou pas correctement installé. Veuillez exécuter 'yunohost tools postinstall'",
+    "yunohost_not_installed": "YunoHost n'est pas correctement installé. S'il vous plaît exécuter 'yunohost tools postinstall'",
```

La nouvelle version est bizarre ...  "S'il vous plaît exécutez' n'est pas français ...

```suggestion
    "yunohost_not_installed": "YunoHost n'est pas correctement installé.  Veuillez exécuter 'yunohost tools postinstall'",
```